CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
   - Features Added
   - Bugs Fixed
 
+## 0.10.2
+  - Features Added
+    - Fix bug when entering numbers for uint256 data types in the GenericScheme new proposal modal 
+
+  - Bugs Fixed
+
```

Comment:
how about the animation pr ?

Reply:
https://github.com/daostack/alchemy/pull/1526